wp_api/src/wp_com/support_bots.rs

```diff
@@ -0,0 +1,187 @@
+use crate::{
+    date::WpGmtDateTime,
+    url_query::{AppendUrlQueryPairs, QueryPairs, QueryPairsExtension},
+    users::UserId,
+};
+use serde::{Deserialize, Serialize};
+use serde_repr::*;
+use std::collections::HashMap;
+
+use super::WpComSiteId;
+
+#[derive(Debug, PartialEq, Eq, Serialize, uniffi::Record)]
+pub struct CreateBotConversationParams {
+    pub message: String,
+    pub user_id: UserId,
```

Comment:
❓ I wonder if we should have a `WpComUserId` type or if it's OK to keep using the same `UserId` type for both `wp_api` and `wp_com`.

Reply:
They're both a u64 under the hood, so I don't see any benefit to having separate types (for the moment).

It also wouldn't be a breaking change (at least for Rust / Swift) to adjust it later if we find out we missed something